src/inc/corinfo.h

```diff
@@ -1369,6 +1369,9 @@ struct CORINFO_RUNTIME_LOOKUP
     // 1 means that value stored at second offset (offsets[1]) from pointer is offset2, and the next pointer is
     // stored at pointer+offsets[1]+offset2.
     bool                indirectSecondOffset;
+
+    // Dictionary slot index result for a dictionary lookup
+    WORD                slot;
```

Comment:
This is changing JIT/EE interface. Why does the JIT need to know the slot?

Reply:
The JIT/EE doesn't really need this. The reason I made this change was because the CORINFO_RUNTIME_LOOKUP structure was being used to pass all the results of the operation, except for the actual slot, which had to be another extra parameter to the API. So my reason was purely around simplifying the API and reducing the number of out-parameters.

But I see your point, we shouldn't change the jit/ee interface if the jit doesn't need this data :). I'll revert this.